doc/symfony-cache-configuration.rst

```diff
@@ -129,6 +129,29 @@ other hosts, provide the IPs of the machines allowed to purge, or provide a
 RequestMatcher that checks for an Authorization header or similar. *Only set
 one of ``client_ips`` or ``client_matcher``*.
 
+Symfony's `HttpCache` does not support cache invalidation by tags by default.
+However, this library ships with a `TaggableStore` that provides exactly that.
+If you want to use this, adjust your `AppCache` as follows::
+
+    use FOS\HttpCache\SymfonyCache\TaggableStore();
+
+    // ...
+
+    /**
+     * Overwrite constructor to register the TaggableStore.
+     */
+    public function __construct(
+        HttpKernelInterface $kernel,
+        StoreInterface $store,
```

Comment:
can we remove this parameter? its never used as you then create the store manually.

Reply:
or alternatively make $store optional and create it below if not specified.

Reply:
Yeah well, these parameters depend on whether you use the `HttpCache` of the `FrameworkBundle` or `HttpKernel` directly. Not sure what to do here best but I mean, it's kind of obvious what you have to do there, isn't it? 😄

Reply:
$store makes sense for normal constructor but not for this class, because its unused as you define $store below in the constructor code.